src/js/mixins/TabsMixin.js

```diff
@@ -77,34 +80,37 @@ const TabsMixin = {
    * Will return an array of tabs to be rendered.
    * Will only have onClick handlers, as these tabs are not routable.
    *
-   * @param  {String} customClasses to be added to the active component
+   * @param  {Object} props to be added to the active component
    * @return {Array} of tabs to render
    */
-  tabs_getUnroutedTabs(customClasses) {
+  tabs_getUnroutedTabs(props) {
     return TabsUtil.getTabs(
       this.tabs_tabs,
       this.state.currentTab,
-      this.tabs_getUnroutedItem.bind(this, customClasses)
+      this.tabs_getUnroutedItem.bind(this, props)
     );
   },
 
   /**
    * Returns a Link for a given tab. This tab is expected to be routable.
    *
-   * @param  {String} customClasses to be added to the Link item
+   * @param  {Object} props to be added to the Link item
    * @param  {String} tab key of tab to render
    * @return {Component} React component to render
    */
-  tabs_getRoutedItem(customClasses, tab) {
+  tabs_getRoutedItem(props = {}, tab) {
+    let attributes = Object.assign({}, props);
```

Comment:
we can just do `Util.omit(props, ['classNames'])` and it'll do the copying + deleting for us